src/functions/fetchActiveWallet.ts

```diff
@@ -0,0 +1,131 @@
+import { decResLengths, EMPTY_WALLET_UID } from '../constants';
+import {
+  decryptResponse,
+  encryptRequest,
+  request,
+} from '../shared/functions';
+import {
+  validateActiveWallets,
+  validateSharedSecret,
+  validateUrl,
+} from '../shared/validators';
+
+/**
+ * Fetch the active wallet in the device.
```

Comment:
Would add:

> The Lattice has two wallet interfaces: internal and external. If a SafeCard is inserted and unlocked, the external interface is considered "active" and this will return the wallet info for that. Otherwise it will return the info for the internal Lattice wallet.

Reply:
Added 10eb9e0cd49a54c754f287b74e8e74d4b98cb628